local-grapl.env

```diff
@@ -60,10 +60,10 @@ AWS_REGION=us-east-1
 LOCALSTACK_HOST=aws.grapl.test
 DGRAPH_HOST=dgraph.grapl.test
 GRAPL_API_HOST=api.grapl.test
+KAFKA_BROKER_HOST=kafka.grapl.test
```

Comment:
The host variables may no longer be necessary with consul connect. I may remove them, after I take a quick look at Max's work to expose dgraph